joern-cli/frontends/c2cpg/src/main/scala/io/joern/c2cpg/utils/GccSpecificExternalCommand.scala

```diff
@@ -11,7 +11,7 @@ object GccSpecificExternalCommand {
   private val IsWin = scala.util.Properties.isWin
 
   def run(command: Seq[String], cwd: String, extraEnv: Map[String, String] = Map.empty): Try[Seq[String]] = {
-    ExternalCommand.run(command, cwd, mergeStdErrInStdOut = true, extraEnv) match {
+    ExternalCommand.run(command, Some(cwd), mergeStdErrInStdOut = true, extraEnv) match {
```

Comment:
Just to avoid the `Some(null)` edge case
```suggestion
    ExternalCommand.run(command, Option(cwd), mergeStdErrInStdOut = true, extraEnv) match {
```